doc/api/report.md

```diff
@@ -503,7 +503,7 @@ containing `libuv` handle information and an OS platform information section
 showing CPU and memory usage and system limits. An example report can be
 triggered using the Node.js REPL:
 
-```raw
+```text
```

Comment:
IMO, this should be `console`.

Reply:
Thanks. Updated in https://github.com/nodejs/node/pull/33028/commits/2fdcd07821088b647bd6cc92732365ee269e2d26

Reply:
Going to PR normalizing the rest of the info strings. This would be handled by the one for the `console` info strings. Sorry about that, impossible to foresee, but it would be best to not change it in this PR.